timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TimeLockTestUtils.java

```diff
@@ -76,6 +76,7 @@ static TransactionManagerContext createTransactionManager(
                                 .servers(serverUris)
                                 .sslConfiguration(SslConfiguration.of(Paths.get("var/security/trustStore.jks")))
                                 .build())
+                        .client(Optional.of(agent))
```

Comment:
Did we need this? I think it would be a break if it turned out that we _must_ specify a client

Reply:
So, the problem is highlighted by this line - `config().timelock().map(TimeLockClientConfig::getClientOrThrow)`. If timeLockConfig is present then client must be present otherwise, if we try to fetch the client, we throw this SafeIllegalStateEx - `"Tried to read a client from a TimeLockClientConfig, but it hadn't been initialised."`

Reply:
Ah, this is a break then. Can you see a way around this?